lib/kadm5/chpass_s.c

```diff
@@ -67,6 +68,22 @@ change(void *server_handle,
     if (ret)
 	goto out2;
 
+    for (i = 0; i < context->num_hooks; i++) {
```

Comment:
The calling of the hooks could be abstracted out.

Reply:
Ack.